src/layouts/Home.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+import Main from "../components/Main";
+
+const Home = ({ children }) => <Main>{children}</Main>;

```

Comment:
Why do you have so many wrapper components? This one, and for example components/Main, they are doing the same.

Reply:
this is a template architecture, not my.